src/Robo/Commands/Saml/SimpleSamlPhpCommand.php

```diff
@@ -0,0 +1,230 @@
+    }
+
+    return $result;
+  }
+
+  /**
+   * Copies the configuration templates from the library to the project root.
+   *
+   * @return \Robo\Result
+   *
+   * @command simplesamlphp:config:init
+   */
+  public function simpleSamlPhpConfigInit() {
+    $destinationDirectory = "{$this->repoRoot}/simplesamlphp/config";
+
+    $this->say("Copying config files to ${destinationDirectory}.");
+    $result = $this->taskFileSystemStack()
+      ->copy("{$this->repoRoot}/vendor/simplesamlphp/simplesamlphp/config-templates/authsources.php", "${destinationDirectory}/authsources.php")
+      ->copy("{$this->repoRoot}/vendor/simplesamlphp/simplesamlphp/config-templates/config.php", "${destinationDirectory}/config.php")
+      ->copy("{$this->bltRoot}/scripts/simplesamlphp/acquia_config.php", "${destinationDirectory}/acquia_config.php")
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+    $result = $this->taskWriteToFile("{$this->repoRoot}/simplesamlphp/config/config.php")
+      ->text("include 'acquia_config.php';")
+      ->append()
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+
+    $this->say("Copying config files to {$this->repoRoot}/simplesamlphp/metadata.");
+    $result = $this->taskFileSystemStack()
+      ->copy("{$this->repoRoot}/vendor/simplesamlphp/simplesamlphp/metadata-templates/saml20-idp-remote.php", "{$this->repoRoot}/simplesamlphp/metadata/saml20-idp-remote.php")
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+
+    return $result;
+  }
+
+  /**
+   * Copies customized config files into the library on deployments.
+   *
+   * @return \Robo\Result
+   *
+   * @command simplesamlphp:deploy:config
+   */
+  public function simpleSamlPhpDeployConfig() {
+    $this->say('Copying config files to the appropriate place in simplesamlphp library in the deploy artifact.');
+    $result = $this->taskCopyDir(["{$this->repoRoot}/simplesamlphp" => "{$this->repoRoot}/deploy/vendor/simplesamlphp/simplesamlphp"])
+      ->overwrite(TRUE)
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+
+    $result = $this->taskFileSystemStack()
+      ->copy("{$this->bltRoot}/scripts/simplesamlphp/gitignore.txt", "{$this->repoRoot}/deploy/vendor/simplesamlphp/simplesamlphp/.gitignore", TRUE)
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+
+    return $result;
+  }
+
+  /**
+   * Sets value in project.yml to let targets know simplesamlphp is installed.
+   *
+   * @return \Robo\Result
+   */
+  protected function setSimpleSamlPhpInstalled() {
+    $composerBin = $this->getConfigValue('composer.bin');
+    $projectConfigFile = $this->getConfigValue('blt.config-files.project');
+    $this->say("Updating ${projectConfigFile}...");
+    $result = $this->taskExec("{$composerBin}/yaml-cli update:value")
```

Comment:
Hmm would that look like? The classes I see provided by grasmash/yaml-cli are all command files with protected methods (unless I'm looking in the wrong place).